### PR TITLE
Added HMRC Unaudited Dormant LLP validations

### PR DESCRIPTION
**Pull request by allenbross-wf**
#### Description of change

Added HMRC validations for Unaudited Dormant LLPs

#### Steps to Test

CI

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: arelle/plugin/validate/HMRC/ValidateHmrc.py [hunk 1]
```diff
@@ -91,13 +100,73 @@
...
+            (
+                '"Wedi\'i eithrio" or "Eithriad", then "adran 477 o Ddeddf Cwmnïau 2006", then "Partneriaeth Atebolrwydd Cyfyngedig" OR "PAC"',
+                re.compile(r".*(Wedi'i eithrio|Eithriad).*adran 477 o Ddeddf Cwmnïau 2006.*(Partneriaeth Atebolrwydd Cyfyngedig|PAC).*"),
+            ),
+        ),
+    },
+    LP_SEC_480: {
+            'StatementThatCompanyEntitledToExemptionFromAuditUnderSection480CompaniesAct2006RelatingToDormantCompanies': (
+                (
+                    '"Exempt" or "Exemption", then  "section 480 of the Companies Act 2006", then "Limited Liability Partnership" OR "LLP"',
```
Comment (brettkail-wk) [inline arelle/plugin/validate/HMRC/ValidateHmrc.py:143]: Is the double-space in `then  "section` correct?

File: arelle/plugin/validate/HMRC/__init__.py [hunk 2]
```diff
@@ -456,8 +456,9 @@ def checkFacts(facts):
                     _("Element %(elt)s style attribute has disallowed image reference: %(styleImage)s."),
                     modelObject=elt, elt=elt.tag.rpartition("}")[2], styleImage=match)
 
-    hmrc = ValidateHmrc(modelXbrl)
-    hmrc.validate()
+    if val.txmyType != "charities":
+        hmrc = ValidateHmrc(modelXbrl)
+        hmrc.validate()
```
Comment (derekgengenbacher-wf) [inline arelle/plugin/validate/HMRC/__init__.py:461]: Im confused by this change?

File: arelle/plugin/validate/HMRC/ValidateHmrc.py [hunk 3]
```diff
@@ -91,13 +100,73 @@
                 re.compile(r".*Paratowyd yn unol â.*darpariaethau.*cwmnïau bach.*"),
             ),
         ),
-    }
+    },
+    LP_ABRID: {
```
Comment (derekgengenbacher-wf) [inline arelle/plugin/validate/HMRC/ValidateHmrc.py:104]: I see LP.Abrid, but not Co.Abrid? That also should be added even if its not being used directly in this PR.

File: arelle/plugin/validate/HMRC/__init__.py (hunk 2 above)
Comment (allenbross-wf) [reply arelle/plugin/validate/HMRC/__init__.py:461]: All of the validations written so far have the explicit statement:  Taxonomy used must be FRS Taxonomy not FRS Charities

**Review by derekgengenbacher-wf — APPROVED** [review]

**Pull request merged**